mindjunkies/courses/tests/test_views.py

```
@@ -1,9 +1,9 @@
 import pytest
-from model_bakery import baker
-from django.urls import reverse
 from categories.models import Category
+from django.urls import reverse
+from model_bakery import baker
 
-from mindjunkies.courses.models import Course, CourseToken, Rating, Enrollment, LastVisitedCourse, CourseCategory
+from mindjunkies.courses.models import Course, CourseCategory, CourseToken, Enrollment, LastVisitedCourse, Rating
```

Ruff (0.8.2)

`mindjunkies.courses.models.LastVisitedCourse` imported but unused

Remove unused import: `mindjunkies.courses.models.LastVisitedCourse`

(F401)